docs/_articles/en/v3.x/salesforce-graph-engine/rules.md

```diff
@@ -79,15 +79,15 @@ Graph Engine ran into an error while walking this path. Manually verify that you
 - [Filter SOQL Queries Using WITH SECURITY_ENFORCED](https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_classes_with_security_enforced.htm)
 - [Frequently Asked Questions](./en/v3.x/faq/)
 
-## UnusedMethodRule
+## UnusedMethodRule#
 
-UnusedMethodRule detects methods contained in your code that aren’t invoked. At this stage, it detects:
+UnusedMethodRule detects methods contained in your code that aren’t invoked. It detects:
 
-- unused private instance methods
-- unused private constructors
-- unused protected constructors
+- static methods
+- private instance methods
+- constructors
 
-To invoke UnusedMethodRule, you must provide `--engine sfge` and `--projectdir/-p` on `scanner:run`.
+To invoke the non-data- flow analysis rules–UnusedMethodRule and UnimplementedTypeRule–run: `scanner:run --engine sfge --projectdir MyDirectory`.
```

Comment:
Can we say, "To invoke Graph-based rules"? The hyphens may need additional formatting.
For the command, can we use [a fenced code block](https://www.markdownguide.org/extended-syntax/#fenced-code-blocks)?
Here's a more complete version of the command:
```
sfdx scanner:run --engine sfge --projectdir /project/dir --target /project/dir/target
```